docs/multicluster/policy-only-mode.md

```diff
@@ -0,0 +1,80 @@
+# Antrea Multi-cluster with NetworkPolicy Only Mode
+
+Multi-cluster Gateway works with Antrea `networkPolicyOnly` mode, in which
+cross-cluster traffic is routed by Multi-cluster Gateways of member clusters,
+and the traffic goes through Antrea overlay tunnels between Gateways and local
+cluster Pods. Pod traffic within a cluster is still handled by the primary CNI,
+not Antrea.
+
+## Deploying Antrea in `networkPolicyOnly` mode with Multi-cluster feature
+
+This section describes steps to deploy Antrea in `networkPolicyOnly` mode
+with the Multi-cluster feature enabled on an EKS cluster. 
+
+You can follow [the EKS documentation](https://docs.aws.amazon.com/eks/latest/userguide/create-cluster.html)
+to create an EKS cluster, and follow the [Antrea EKS installation guide](./../eks-installation.md)
+to deploy Antrea to an EKS cluster. Please note there are a few changes required
+by Antrea Multi-cluster. You should set the following configuration parameters in
+ `antrea-agent.conf` of the Antrea deployment manifest to enable the `Multicluster`
+ feature and Antrea Multi-cluster Gateway:
+
+```yaml
+antrea-agent.conf: |
+...
+  featureGates:
+...
+    Multicluster: true
+...
+  multicluster:
+    enableGateway: true
+    namespace: "" # Change to the Namespace where antrea-mc-controller is deployed.
+```
+
+Antrea in a single cluster will be ready to handle cross-cluster traffic.
```

Comment:
I suggest to remove this sentence. It just sounds unrelated here, after the YAML.

Reply:
Removed